test-app/config/ember-try.js

```diff
@@ -7,14 +7,6 @@ module.exports = async function () {
   return {
     usePnpm: true,
     scenarios: [
-      {
```

Comment:
I removed the 3.28 support because i got errors from ember-source@3.28 on bowerDependencies (and because it's an old lts). I did not investigate further. But if you still want to keep the 3.x compatibility i can look further into it.